### PR TITLE
Adding dummy_id primary key columns

### PR DESCRIPTION
**Pull request by hcli-work**
I'm not sure if this is what you were envisioning. I created a directory to put the SQL scripts. One does the changes to an existing kits database. The other reverts those changes just in case something goes south.

I also put the changes in attendance.php as per Adam Ruppe's request.

### CODE REVIEW

**Comment by sadleb** [comment]
This is a nit-picky request, but can we mimic the folder structure of a rails app for these scripts? e.g. create a db/migrate folder and put the scripts in there?

File: attendance.php [hunk 1]
```diff
@@ -55,6 +55,30 @@
...
+	      ADD dummy_id INT PRIMARY KEY AUTO_INCREMENT,
+	      ADD UNIQUE INDEX unique_index (`event_id`, `person_id`);
+
+	USE braven_attendance;
+	ALTER TABLE attendance_lc_absences
+	      DROP PRIMARY KEY,
+	      ADD dummy_id INT PRIMARY KEY AUTO_INCREMENT,
+	      ADD UNIQUE INDEX unique_index (`event_id`, `lc_email`);
+
+	USE wordpress;
```
Comment (adamdruppe) [inline attendance.php]: the wordpress stuff wouldn't be in attendance.php, this file is just for the attendance specific bit. so can remove this

File: attendance.php [hunk 2]
```diff
@@ -55,6 +55,30 @@
 		PRIMARY KEY(id)
 	) DEFAULT CHARACTER SET=utf8mb4;
 
+	USE braven_attendance;
+	ALTER TABLE attendance_people
```
Comment (adamdruppe) [inline attendance.php]: and what i had in mind for this was it wouldn't be alter, but just the create things. I basically use it to at-a-glance see what is supposed to be there and having to scroll up and down to see create vs alter defeats that.

but i can make that edit myself too

**Comment by hcli-work** [comment]
Can do regarding db/migrate
I'll rip out the wordpress specific things and also see what the resulting CREATE TABLE statement would look like.

**Pull request merged**